src/original-content/OriginalGameFiles.cpp

```diff
@@ -60,6 +65,11 @@ namespace REGoth
     return filterFilesInDirectoryByExt(dataDirectory(), ".vdf");
   }
 
+  bs::Vector<bs::Path> OriginalGameFiles::allModPackages() const
+  {
+    return filterFilesInDirectoryByExtRecursive(modDirectory(), ".mod");
```

Comment:
`*.mod` or `*.modvdf`? The documentation of `loadGamePackages` says `*.modvdf`

Reply:
`.mod` is correct. Good catch.